packages/theme/template/package.json

```diff
@@ -28,20 +28,20 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.6.0",
-    "@enact/i18n": "^4.6.0",
-    "@enact/spotlight": "^4.6.0",
-    "@enact/ui": "^4.6.0",
+    "@enact/core": "^4.6.2",
+    "@enact/i18n": "^4.6.2",
+    "@enact/spotlight": "^4.6.2",
+    "@enact/ui": "^4.6.2",
     "classnames": "^2.3.2",
     "prop-types": "^15.8.1",
-    "ramda": "^0.28.0",
+    "ramda": "^0.29.0",
     "react": "^18.2.0"
   },
   "peerDependencies": {
     "ilib": "^14.15.1 || ^14.15.1-webos1"
```

Comment:
please update this dependency also
ilib versions: https://www.npmjs.com/package/ilib?activeTab=versions
ilib-webos versions: https://www.npmjs.com/package/ilib-webos?activeTab=versions